ArjunaJTA/jta/classes/com/arjuna/ats/jta/common/JTAEnvironmentBean.java

```diff
@@ -113,7 +115,11 @@
 
 	private Map<String, Integer> commitMarkableResourceRecordDeleteBatchSizeMap = new HashMap<String, Integer>();
 
-	private static final String defaultTransactionOperationsProviderClassName = ServerVMClientUserTransactionOperationsProvider.class.getName(); 
+	private static final String defaultSubordinateTransactionImporterClassName = NarayanaSubordinateTransactionImporter.class.getName();
```

Comment:
I imagine it looks strange because of the way the diff looks like @ochaloup removed and added other stuff but I believe it is correct how eb8239b477cb1fb32e55c2cf489fbed1c84cbb9d renders. Is that right? You didn't actually remove anything in eb8239b477cb1fb32e55c2cf489fbed1c84cbb9d?